### ISSUE TITLE
Vue.version is undefined

### ISSUE THREAD

**Issue by feitian124** [parcel-bundler/examples#20]
run example ok, the console output is as below:
```bash
± |master ✓| → yarn start
yarn run v1.7.0
warning package.json: No license field
$ parcel
Server running at http://localhost:1234 
✨  Built in 1.20s.
```
but the html page is blank, and can see error in console as below:

Vue.version is undefined.

**Comment by punkkk** [comment]
Add 'index.html' to 'start' script. It would help.

**Comment by punkkk** [comment]
It is probably fixed wit [this pr](https://github.com/parcel-bundler/examples/pull/22)

**Comment by jthegedus** [comment]
closed in #22
